scene/gui/graph_node.cpp

```diff
@@ -669,6 +672,11 @@ void GraphNode::_port_pos_update() {
 			if (slot_table[slot_index].enable_left) {
 				PortCache port_cache;
 				port_cache.pos = Point2i(edgeofs, vertical_ofs + size.height / 2);
+				// Apply rotation.
+				Vector2 rotated_pos = Vector2(
+						port_cache.pos.x * cos_rot - port_cache.pos.y * sin_rot,
+						port_cache.pos.x * sin_rot + port_cache.pos.y * cos_rot);
+				port_cache.pos = rotated_pos;
```

Comment:
```suggestion
				port_cache.pos = port_cache.pos.rotated(get_rotation());
```
I'd suggest using this instead of computing it yourself

Reply:
that means removing everything else i wrote and replace it with only that ?

Reply:
This is the code:
https://github.com/godotengine/godot/blob/4ab8fb809396fa38ba929fec97cfcb7193f1c44d/core/math/vector2.cpp#L112-L118

Reply:
Thanks for the clarification and help.